silkworm/rpc/core/evm_executor.hpp

```diff
@@ -41,11 +41,19 @@
 
 namespace silkworm::rpc {
 
+enum PreCheckErrorCode {
+    EC_INTRINSIC_GAS_TOO_LOW,
```

Comment:
`EC_INTRINSIC_GAS_TOO_LOW ` -> `kIntrinsicGasTooLow` and so on as per [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Enumerator_Names)

Reply:
ok